lib/srv/authhandlers.go

```diff
@@ -348,9 +348,10 @@ func (h *AuthHandlers) canLoginWithRBAC(cert *ssh.Certificate, clusterName strin
 	if err != nil {
 		return trace.Wrap(err)
 	}
+	_, mfaVerified := cert.Extensions[teleport.CertExtensionMFAVerified]
```

Comment:
I assume it can never have "false" value i.e. the presence of this extension always indicates MFA is verified? Or would it be a bit safer to explicitly check if the value is "true" (or whatever gets set there)?

Reply:
All the flag extension values are empty: https://github.com/gravitational/teleport/blob/0f49d601a78869d3bf5c5be2511449588b41bfac/lib/auth/native/native.go#L271-L282
Only their presence matters.